test/test-raja-perf-suite.cpp

```diff
@@ -32,27 +32,38 @@ TEST(ShortSuiteTest, Basic)
   argc = 7;
 #endif
 
+  char *unit_test = getenv("RAJA_PERF_UNIT_TEST");
+  if (unit_test != NULL)
+    argc += 2;
+
+  int next_index = 0;
+
   std::vector< std::string > sargv(argc);
-  sargv[0] = std::string("dummy ");  // for executable name
-  sargv[1] = std::string("--checkrun");
-  sargv[2] = std::string("3");
-  sargv[3] = std::string("--show-progress");
-  sargv[4] = std::string("--disable-warmup");
+  sargv[next_index++] = std::string("dummy ");  // for executable name
```

Comment:
As long as we're thinking about changing this we should use c++ style `emplace_back` to add to `sargv` and let `argc = sargv.size()` at the end.

Reply:
👍